### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xcud**
Adds the following events or companies:
 - Positronic AI

### Checklist

 - [X] I have updated the event or company counts
 - [X] I have put the most recent relevant date in the "Last Update" column
 - [] I have linked to the article about the change, not the company's website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | PingCap | Required for Bay Area | Restricted | Restricted | Restricted | 2020-03-07 |
 | Pinterest | Encouraged | Restricted | Restricted | Partially Restricted | 2020-03-08 |
 | Polidea | Encouraged | Restricted | Restricted | Restricted | 2020-03-10 |
+| Positronic AI | Required | Restricted | Restricted | Restricted | 2020-03-11 |
```
Comment (drewbrew) [inline README.md]: Do you have a public-facing post for this change by chance?

Also, would you mind updating the count?

Comment (xcud) [reply README.md]: Rebased from most current master. Updated count. Official announcement was over (internal) Slack yesterday.
image

Will a tweet do? https://twitter.com/positronicAI/status/1237953785999306752

Comment (drewbrew) [reply README.md]: The tweet is great!

**Comment by drewbrew** [comment]
We've switched to a YAML-based approach overnight, so once you update to use that format (might be easier to start a new PR, honestly), I'll be happy to merge.

Thanks!

**Comment by phildini** [comment]
Closing due to rebase and staleness. Feel free to re-open after fixes, thanks for your patience.

**Pull request closed without merging**